frontend/src/static/js/v_authorship.js

```diff
@@ -121,7 +121,7 @@ window.vAuthorship = {
       }
 
       if (!this.info.fileTypeColors) {
-        this.$root.$emit('restoreFileTypeColors', this.info);
+        this.info.fileTypeColors = this.$store.state.fileTypeColors;
```

Comment:
![image](https://user-images.githubusercontent.com/35621726/92097788-0c803080-ee0b-11ea-9eac-847e56f407ec.png)

I believe there is no need to store fileTypeColors in the info anymore. Let's set it to a computed map state value just like how we do it in `vSummaryCharts`